lib/Command/Room/Create.php

```diff
@@ -105,11 +105,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		$owner = $input->getOption('owner');
 		$moderators = $input->getOption('moderator');
 
-		if (!in_array($readOnly, [null, (string)Room::READ_WRITE, (string)Room::READ_ONLY], true)) {
```

Comment:
Why is this?

Reply:
On interaction on cli, this value never will be different of boolean because this option have the mode `InputOption::VALUE_NONE` and when the interaction is from cli, the value will be boolean.
Search by `InputOption::VALUE_NONE` on this page: https://symfony.com/doc/current/console/input.html

I believe that this conditional  is implemented because when we run unit tests is possible define a value different of boolean because the unit tests don't pass by validation types of options.

Reply:
And the variable name is wrote using O (uppercase), different of the name defined on argument of method. This will generate a `Undefined variable` warning and never will enter on `if`.

Reply:
🙈